clients/cli/src/main/java/org/apache/gravitino/cli/outputs/TableFormat.java

```diff
@@ -79,6 +81,19 @@ public void output(Catalog catalog) {
     }
   }
 
+  static final class CatalogsTableFormat implements OutputFormat<Catalog[]> {
+    @Override
+    public void output(Catalog[] catalogs) {
+      List<String> headers = Collections.singletonList("catalog");
+      List<List<String>> rows = new ArrayList<>();
+      for (int i = 0; i < catalogs.length; i++) {
+        rows.add(Arrays.asList(catalogs[i].name()));
```

Comment:
I think we can display more filed values, just like `type()`, `provider()`, `comment` and `properties()`.

Reply:
That is what would be displayed in a details command not a list command.

Reply:
I agreed.